deployment/media-insights-stack.yaml

```diff
@@ -225,6 +235,11 @@ Resources:
                   - "Fn::GetAtt":
                       - WorkflowExecutionLambdaDeadLetterQueue
                       - Arn
+              - Effect: Allow
+                Action:
+                  - "xray:PutTraceSegments"
+                  - "xray:PutTelemetryRecords"
+                Resource: "*"
```

Comment:
Does this have to be *? If it does, might be worth it to add a cfn_nag exception so we don't get flagged in the SB pipeline.

Reply:
As far as I can tell, yes.  If you try using the IAM console to create this permission, it does not give any option to enter a specific resource.  It says "the actions you chose support all resources".

![image](https://user-images.githubusercontent.com/33331299/96290624-c71f5900-0f9b-11eb-9cd7-0f39a42dedd5.png)

Reply:
If you are OK with this, can you point me to an example of the cfn nag I need to add

Reply:
WARN W11 is the message you'll need to add an exception for. I'll DM you an example.